common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/protocol/FetchShuffleBlocks.java

```diff
@@ -36,19 +36,22 @@
   // it corresponds to the i-th int[] in reduceIds, which contains all reduce id for this map id.
   public final int[] mapIds;
   public final int[][] reduceIds;
+  public final int shuffleGenerationId;
```

Comment:
nit: can we move it just after `public final int shuffleId;`?

Reply:
Copy that, changed all order in 5a4b8b8.